BFWControls/Modules/Table/Controller/StaticTableViewController.swift

```diff
@@ -114,9 +99,21 @@ class StaticTableViewController: UITableViewController {
     // MARK: - UITableViewDelegate
 
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        let height = intrinsicHeightCells
+        var height = intrinsicHeightCells
             ? UITableViewAutomaticDimension
             : super.tableView(tableView, heightForRowAt: superIndexPath(for: indexPath))
+        // Check filledUsingLastCell attribute.
+        if filledUsingLastCell {
+            // Is this last cell?
+            if indexPath == lastCellIndexPath {
+                let previousRowFrame = tableView.rectForRow(at: IndexPath(row: indexPath.row-1, section: indexPath.section))
```

Comment:
Please insert a space of both sides of the `-` operator, like you have for the `+` below.